formats/opentype.snap

```diff
@@ -16,8 +16,12 @@ def find_table : fun (num_tables : U16) -> Array16 num_tables {
     checksum : U32,
     offset : U32,
     length : U32,
-} =
-fun num_tables table_records table_id => array16_find num_tables (Repr table_record) (fun table_record => table_record.table_id == table_id) table_records;
+} = fun @num_tables table_records table_id => array16_find @num_tables @{
+    table_id : U32,
+    checksum : U32,
+    offset : U32,
+    length : U32,
+} (fun table_record => table_record.table_id == table_id) table_records;
```

Comment:
Note that this results bloated terms in the elaborated output. I’d imagine that glued evaluation _might_ be able to keep these terms small, but I could be wrong.